src/Microsoft.Health.Fhir.CosmosDb/Microsoft.Health.Fhir.CosmosDb.csproj

```diff
@@ -13,6 +13,7 @@
     <PackageReference Include="MediatR.Extensions.Microsoft.DependencyInjection" Version="8.1.0" />
     <PackageReference Include="Microsoft.Azure.Cosmos" Version="3.12.0" />
     <PackageReference Include="Microsoft.Azure.Cosmos.Direct" Version="3.4.1.0" /> <!-- Workaround for https://github.com/dotnet/roslyn/issues/47304 when building in VS-->
+    <PackageReference Include="System.Private.ServiceModel" Version="4.5.3" /><!-- Microsoft.Azure.Cosmos.Direct is referencing an insecure version of System.Private.ServiceModel (4.5.0) -->
```

Comment:
Is there a way to flag this for future review? If we don't normally need this for our project it would be nice to not have to worry about it once Cosmos Direct updates.

Reply:
Hopefully we should see it if the line above is updated so we can check again :)